apis/cf/latest/paths/Spaces.yaml

```diff
@@ -153,6 +153,14 @@
     parameters:
       - $ref: '../components/parameters/Guid.yaml'
     responses:
+      '201':
```

Comment:
Curious about some of these results from wiretap, I don't see how CC returned 201 for a space delete

https://github.com/cloudfoundry/cloud_controller_ng/blob/main/app/controllers/v3/spaces_controller.rb#L96